src/presentational/navigation.component.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Link } from "react-router";
+import "../../node_modules/bootstrap/dist/css/bootstrap.css";
+import "../../node_modules/bootstrap/dist/css/bootstrap-theme.css";
+
+const Navigation = props => (
+  <div>
+    <nav className="navbar navbar-default">
+      <div className="container-fluid">
+        <div className="navbar-header">
+          <Link className="navbar-brand" to="/">
+            Countries
+          </Link>
+        </div>
+        <div className="collapse navbar-collapse">
+          <ul className="nav navbar-nav">
+            <li>
```

Comment:
you can create list elements using some of array functions - for example forEach or map ;)